### DISCUSSION TITLE
Clarification required regarding connaisseur's support policy

### DISCUSSION THREAD

**Discussion by ishaqkhattana** [sse-secure-systems/connaisseur#1228] [General] [answered]
Hey 👋  
As per the official documentation, specifically [this line](https://sse-secure-systems.github.io/connaisseur/v3.0.0/SECURITY/#supported-versions:~:text=At%20present%2C%20we%20only%20support%20the%20latest%20version), Connaisseur claims to support the latest version. I wanted to get clarity on what 'version' means here. Is it the latest major, major.minor or the major.minor.patch version?

**Comment by phbelitz** [comment, marked as answer]
Hoy 👋 
Strictly speaking it's `major.minor.patch`, which doesn't mean we won't help you if you have issues with an older version. But if you have an issue with an older version and we need to make fixes, those fixes will be deployed to the newest version, not the one you have, so you need to update eventually. Bugs you find in Connaisseur 2.X will be fixed in 3.X :shrug: Does that answer you question?

**Reply by ishaqkhattana** [reply]
It does, thanks a lot 🙏
